### ISSUE TITLE
Can I include html in the message when using the API? is that supported?

### ISSUE THREAD

**Issue by mauriciojsola** [aslanon/vue-confirm-dialog#15]
Can I include html in the message when using the API? is that supported?

**Comment by aslanon** [comment]
No, that is not supported. But I can add this feature to the next release.

**Comment by mauriciojsola** [comment]
That would be great. Thanks!

**Comment by podolskyi** [comment]
@aslanon yes, please

**Comment by nekooee** [comment]
please add this feature. I want to use icon fonts that need HTML support.

**Comment by LeoLobLogiciel** [comment]
> please add this feature. I want to use icon fonts that need HTML support.

https://www.npmjs.com/package/vue-lsi-util

**Comment by nekooee** [comment]
> > please add this feature. I want to use icon fonts that need HTML support.
> 
> https://www.npmjs.com/package/vue-lsi-util

I did not find any photos or demos of this package. What does it look like?

**Comment by LeoLobLogiciel** [comment]
<img width="976" alt="Captura de Pantalla 2021-03-07 a la(s) 19 13 08" src="https://user-images.githubusercontent.com/69116641/110256727-2f85cb80-7f79-11eb-8444-16a11c7ed8d4.png">

Here is an example.  You CAN put HTML into the message.

**Comment by mkarcsi** [comment]
Not the most beautiful solution, but you can manipulate the DOM of confirm window with JS

After `this.$confirm()` you can add like this:

`setTimeout(() => {
  document.querySelector('.vc-container .vc-text').innerHTML = '<strong>Hello</strong>';
}, 50);`
